### ISSUE TITLE
Raha Utility Function

### ISSUE THREAD

**Issue by rahulgi** [rahafoundation/documentation#6]
Copied from @osdiab's comments on PR #1.

[comment 1](https://github.com/rahafoundation/documentation/pull/1#discussion_r178982040)
"""
So is what you're saying that we're trying to maximize this utility function? I'm a little confused what exactly you're trying to express here, and I also think that this mathematical approach might be appealing to some but doesn't capture what we're qualitatively trying to achieve. Much rather have this more as a footnote than the first, primary way we express our objective.
"""
[comment 2](https://github.com/rahafoundation/documentation/pull/1#discussion_r178982711)
"""
I'm fine with [the utility function] being there as an aside on how we're measuring our progress or under some different label, but it's not a compelling story for what we value and why we're doing this. It's a pretty low appeal and confusing to most people means of expressing what our goals are, and the actual proportions of importance and stuff are largely arbitrary—for instance, logarithms generally scale so far below linear that we could probably maximize this function by largely ignoring the size of the UBI altogether and just getting as many people on the network as we can, like by putting some arbitrary game or social networking element on the platform.
"""

**Comment by markulrich** [comment]
The vibe of a single formula to capture total value being central to our main document seems a bit too absolutist (don't wanna be known as some [paperclip maximizer](https://wiki.lesswrong.com/wiki/Paperclip_maximizer)).

What if we remove the utility function from the primary document (maybe put it under its own wiki page?) unless people will understand it well (which I currently do not - what's purpose of centrally controlled variable `u`?) and we believe it will better help us accomplish our mission (how would this formula change policy, or change member understanding?).

That said, even if we're going for humanitarian [social economy](https://en.wikipedia.org/wiki/Social_economy) vibe, that doesn't mean math and numbers are the enemy. Some data that captures the spirit of "making the wealthy become fabulously wealthy doesn't help society all that much" would be great.

![image](https://user-images.githubusercontent.com/6357579/38348719-42ff0fb6-3858-11e8-9bdd-b55e467e41d7.png)
Gotta admit this plot from http://www.scielo.org.co/scielo.php?script=sci_arttext&pid=S0123-59232013000100002 does look like a log!

**Comment by osdiab** [comment]
Yeah having a data driven approach isn't a bad thing per se, I agree with @markulrich 's assessment of the image it entails and my lack of understanding not only how it was concocted but also the kinds of behavior it would encourage.

**Comment by osdiab** [comment]
As I mentioned in #7 though some form of this can actually be useful as a means of defining payout structure to the organization, because the money we disburse for development ought to be tied to performance specifically in a quantifiable way.
